src/main/java/fr/lunatech/timekeeper/services/responses/EventTemplateResponse.java

```diff
@@ -124,55 +124,5 @@ public String toString() {
                 '}';
     }
 
-    public static final class Attendee {
```

Comment:
I prefer to keep Attendee here, and not as part of the model, as it's a Data transfer object (and not something we really persist)